compiler/cli/src/org/jetbrains/kotlin/cli/jvm/compiler/TopDownAnalyzerFacadeForJVM.kt

```diff
@@ -99,20 +99,21 @@ object TopDownAnalyzerFacadeForJVM {
             return null
         }
 
+        fun getExtensionsResult(): AnalysisResult? {
```

Comment:
I'd prefer moving the line `container.get<JavaClassesTracker>().onCompletedAnalysis(module)` to the beginning of `invokeExtensionsOnAnalysisComplete` since `JavaClassesTracker` is also a kind of extension and `invokeExtensionsOnAnalysisComplete` has the single call-site now

Reply:
Thanks for the comment, it does make it more readable.